arm9/lib/include/userInfo.h

```diff
@@ -0,0 +1,92 @@
+#ifndef POKEDIAMOND_USERINFO_H
+#define POKEDIAMOND_USERINFO_H
+
+typedef struct NVRAMConfigDate
```

Comment:
Put offset comments on each field, and total size comments at the end of a struct definition.

Reply:
Align address comments vertically